js/curseprofile.js

```diff
@@ -55,7 +55,10 @@
 			event.stopPropagation();
 		});
 
-		
+		// Prevent links generated by wikitext from opening a new window
+		$('.user-profile-navigation__link a').each(function () {
+			$(this).removeAttr('target');
```

Comment:
Are there any edge cases where we don't want this?

Reply:
No edge cases, and this will only affect navigation tabs on the Curse Profile and nothing else.